runtime/default/manual_task.c

```diff
@@ -0,0 +1,12 @@
+#include <stdio.h>
+
+void srt_handle_manual_task(const char *element_id, const char *instructions) {
+  printf("Manual Task %s\n", element_id);
+
+  if (instructions != NULL) {
+    printf("  * %s\n", instructions);
+  }
+
+  printf("\nPress enter to continue:\n");
+  getchar();
+}
```

Comment:
Good suggestion, I was trying to do that by requiring `manual_task.c` or `manual_task_headless.c` to be specified at build time, but that is sub-optimal. Added a `isatty` check and updated the Makefile to only allocate a pseudo-TTY when opening a shell into the docker container. This will simplify things.